sqlmesh/core/scheduler.py

```diff
@@ -4,8 +4,8 @@
 import logging
 import traceback
 import typing as t
-from datetime import datetime
 
+from more_itertools import flatten
```

Comment:
this seems like a transitive dep, either don't use it or add it explicitly to setup.py